From dc29a9a30efcd5d21bb7fd491d0a50d1cc9db8b1 Mon Sep 17 00:00:00 2001
From: Carlos Rodriguez <carlos.a.rodriguez@intel.com>
Date: Tue, 1 Mar 2016 01:56:30 -0600
Subject: [PATCH 1/2] ASoC: Intel: Change SMBIOS board name

There was a SMBIOS update and board name changed
from "Cherry Trail MRD" to "T3 MRD".
This patch updates sst driver to reflect the update.

Change-Id: If768bc80e4ac2f195860a94925ad293e24676360
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-22084
Signed-off-by: Carlos Rodriguez <carlos.a.rodriguez@intel.com>
Reviewed-on: https://android.intel.com:443/476408
---
 sound/soc/intel/sst/sst.c          | 2 +-
 sound/soc/intel/sst/sst_platform.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/sst/sst.c b/sound/soc/intel/sst/sst.c
index 480f3d7..10b9c58 100644
--- a/sound/soc/intel/sst/sst.c
+++ b/sound/soc/intel/sst/sst.c
@@ -676,7 +676,7 @@ int sst_request_firmware_async(struct intel_sst_drv *ctx)
 				"fw_sst_%04x.bin", ctx->pci_id);
 
 		board_name = dmi_get_system_info(DMI_BOARD_NAME);
-		if (strcmp(board_name, "Cherry Trail MRD") == 0) {
+		if (strcmp(board_name, "T3 MRD") == 0) {
 			pr_info("Registering machine device %s\n",
 						cht_cr_mrd_mach_dev.name);
 			ret = platform_device_register(
diff --git a/sound/soc/intel/sst/sst_platform.c b/sound/soc/intel/sst/sst_platform.c
index 9bb584d..2ad590d 100644
--- a/sound/soc/intel/sst/sst_platform.c
+++ b/sound/soc/intel/sst/sst_platform.c
@@ -355,7 +355,7 @@ static void set_cht_platform_config(void)
 	memcpy(sst_platform_pdata.ssp_config, sst_ssp_configs_mrfld, sizeof(sst_ssp_configs_mrfld));
 	board_name = dmi_get_system_info(DMI_BOARD_NAME);
 	if ((strcmp(board_name, "Cherry Trail CR") == 0) ||
-			(strcmp(board_name, "Cherry Trail MRD") == 0)) {
+			(strcmp(board_name, "T3 MRD") == 0)) {
 		pr_debug("Load CHT CR SSP Config %s\n", board_name);
 		memcpy(sst_platform_pdata.ssp_config, sst_ssp_configs_cht_cr,
 					sizeof(sst_ssp_configs_cht_cr));
-- 
1.9.1

